sdk/examples/intkey_go/src/sawtooth_intkey/main.go

```diff
@@ -51,6 +52,7 @@ func main() {
 	handler := intkey.NewIntkeyHandler(prefix)
 	processor := processor.NewTransactionProcessor(endpoint)
 	processor.AddHandler(handler)
+	processor.ShutdownOnSignal(syscall.SIGINT, syscall.SIGTERM)
```

Comment:
Is there a reason that this can't be contained within the TransactionProcessor class?